homeassistant/const.py

```diff
@@ -248,6 +247,10 @@
 DEVICE_CLASS_TIMESTAMP: Final = "timestamp"
 DEVICE_CLASS_VOLTAGE: Final = "voltage"
 
+# ### STATE CLASSES ####
+STATE_CLASS_MEASUREMENT: Final = "measurement"
+STATE_CLASS_NONE: Final = "none"
```

Comment:
Please remove this constant. It's not needed. We use `None` to represent no state class set.